Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -825,11 +825,11 @@ class DropDownMenu
     return item;
   }
 
-  private static OnItemClick (this: HTMLAnchorElement): boolean
+  private static OnItemClick (this: HTMLAnchorElement, ev:HTMLElementEventMap["click"]): boolean
   {
+    ev.preventDefault();
```

Comment:
I would like to pull this as a bugfix into a separate PR that gets integrated first and is linked as a blocking issue. The original code didn't result in an obvious problem but it was still incorrect behavior and isn't part of this change. By keeping things strictly separate, looking for bug-causes and why why have been introduced is much easier. Same goes for reasoning over a changeset in general. For instance "why is there more changes in DropDownMenu.ts than there is in ListMenu.ts? -> ah, okay, there was a bugfix that needs to get done. We don't have this bug over there? why? etc etc.